### ISSUE TITLE
ReferenceError: Can't find variable: exports

### ISSUE THREAD

**Issue by millsb** [wallabyjs/public#222]
Trying out wallaby for the first time, and working with ES6 browser-code. Running it through Babel and Browserify, but ending up with this error thrown in each file.

`ReferenceError: Can't find variable: exports`

Seems like somehow Browersify isn't injecting the required globals for CommonJS modules to the output. A sample code file would look like:

```
import { BaseFilter } from './BaseFilter';
import { Param } from '../Param';

export class ValueFilter extends BaseFilter {
    constructor({ id, value }) {
        super();
        this.id = id;
        this.value = value;
    }

    toParam() {
        return new Param({ id: this.id, value: this.value });
    }
}
```

And my config is as follows:

```
module.exports = function(wallaby) {
	var babel = require('babel');
	var babelCompiler = wallaby.compilers.babel({
		babel: babel
	});
	var wallabify = require('wallabify');
	var wallabyPostprocessor = wallabify({});
	return {
		files: [
			'src/**/*.js'
		],
		tests: [
			'test/**/*.js'
		],
		compilers: {
			'**/*.js*': babelCompiler
		},
		postprocessor: wallabyPostprocessor,
		bootstrap: function () {
			// required to trigger tests loading
			window.__moduleBundler.loadTests();
		}
	}
};
```

Not sure what else to try. Thanks!

**Comment by ArtemGovorov** [comment]
@millsb I think you're almost there in terms making it work, just add `load: false` to your file and test patterns like in [this sample config](http://wallabyjs.com/docs/integration/browserify.html#usage)
``` JavaScript
        files: [
            {pattern: 'src/**/*.js', load: false}
        ],
        tests: [
            {pattern: 'test/**/*.js', load: false}
        ],
```

Otherwise (with `load: true` which is the default) it tries to load the files into the sandbox by including them in `script` tags, as opposed to loading them when `window.__moduleBundler.loadTests();` is called, and causing the error.

Please let me know if setting `load: false` helps.

**Comment by millsb** [comment]
That did the trick. Thanks.

One more question. All my tests are running just fine now, except for a suite I'm purposefully skipping. So I see this error:

`TypeError: 'undefined' is not a function (evaluating 'describe.skip')`

I'm coming from Karma, which was injecting my test libs as plugins, and so to get chai and sinon to working again, I had to add them to my bundle configuration and import them per usual as modules. However, I did not have to do that with Mocha. How is `describe` normally resolved for my other tests, but not `skip()`?

Thanks again.
